R/format.R

```diff
@@ -196,7 +196,7 @@ colFormatter = function(name, names, rownames = TRUE, template, ...) {
   js = sprintf("
   function(data, type, row, meta) {
     if (type === 'display') {
-      return %s
+      return %s;
```

Comment:
Haha, it's not missed because the `template(...)` function will include the `;`

Reply:
Oh, sorry about that. In that case, I shouldn't use the ternary operator, either. I'll fix it.

Reply:
Done: bbe6474 (the ternary operator can still be used; I just need to make sure not to add the semicolon)